x-pack/plugin/esql/qa/testFixtures/src/main/resources/docs.csv-spec

```diff
@@ -660,18 +660,19 @@ FROM sample_data
 docsBitLength
 required_capability: fn_bit_length
 // tag::bitLength[]
-FROM employees
-| KEEP first_name, last_name
-| EVAL fn_bit_length = BIT_LENGTH(first_name)
+FROM airports
+| WHERE country == "India"
+| KEEP city
+| EVAL fn_length=LENGTH(city), fn_bit_length = BIT_LENGTH(city)
```

Comment:
I noticed that everywhere else we have whitespace around the `=` sign. In the docs preview this was more obvious.

Reply:
Will fix in follow-up PR.